pnpm-lock.yaml

```diff
@@ -124,6 +124,7 @@ importers:
       jest: ^27.5.1
       jest-cli: ^27.5.1
       jest-mock-extended: ^1.0.18
+      require-turbo: workspace:*
```

Comment:
I rebased from trunk and re-installed the plugin.  These are changes from the original branch, but present in the current trunk.